src/module.c

```diff
@@ -5401,17 +5411,33 @@ int RM_HashGet(RedisModuleKey *key, int flags, ...) {
 
         /* Query the hash for existence or value object. */
         if (flags & REDISMODULE_HASH_EXISTS) {
+            
+            /* Verify flag is not set together with REDISMODULE_HASH_EXPIRE_TIME */
+            if (flags & REDISMODULE_HASH_EXPIRE_TIME)
+                return REDISMODULE_ERR;
+            
             existsptr = va_arg(ap,int*);
             if (key->value) {
                 *existsptr = hashTypeExists(key->db, key->value, field->ptr, hfeFlags, NULL);
             } else {
                 *existsptr = 0;
             }
+        } if (flags & REDISMODULE_HASH_EXPIRE_TIME) {
```

Comment:
why is the `if` together with the `}`? was it meant to be an else-if?

Reply:
forgot `else`